### PR TITLE
feat: #114 cart link in header

### PR DESCRIPTION
**Pull request by plaplante-lab**

### CODE REVIEW

File: src/packages/@cqdg/components/header/Header.jsx [hunk 1]
```diff
@@ -13,6 +13,7 @@ import Link from '@ferlab-ui/core/buttons/link';
 import { withRouter } from 'react-router-dom';
 import { setLanguageAction } from '@cqdg/store/intl';
 import './Header.css';
+import CartIcon from "../icons/CartIcon";
```
Comment (francisl) [inline src/packages/@cqdg/components/header/Header.jsx:16]: absolute import !

File: src/packages/@cqdg/pages/Cart/CartPage.js [hunk 2]
```diff
@@ -0,0 +1,415 @@
...
+        value: files.map(f => f.file_id),
+      })
+    : null;
+
+  return (
+    <Column style={styles.container} className="test-cart-page">
+      {!files.length && <h1>Your cart is empty.</h1>}
+      {!!files.length && (
+        <Column>
+          {/*<Row style={{ marginBottom: '2rem', flexWrap: 'wrap' }}>*/}
```
Comment (francisl) [inline src/packages/@cqdg/pages/Cart/CartPage.js:106]: Do you see value in keeping all these commented lines?

**Review by francisl — COMMENTED** [review]
LGTM, 2 comments

**Review by francisl — APPROVED** [review]
Fix in another PR

File: src/packages/@cqdg/pages/Cart/CartPage.js (hunk 2 above)
Comment (plaplante-lab) [reply src/packages/@cqdg/pages/Cart/CartPage.js:106]: Yes, I'll try to make it work :)

**Pull request merged**